src/breadcrumb/breadcrumb.scss

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-breadcrumb {
+  @include iui-reset;
+  @include iui-font-family;
+  display: flex;
+  align-items: center;
+
+  > .iui-button {
+    margin-right: $iui-s;
+  }
+  
+  ol {
+    @include iui-reset;
+    list-style-type: none;
+    display: flex;
+    align-items: center;
+    user-select: none;
+
+    li {
```

Comment:
Relying on `ol` and `li` will cause problems. I'd say go with class names here

Reply:
Ok, giving classes in the next push.